### PR TITLE
bridge updates

### PR DESCRIPTION
**Pull request by shaspitz**

### CODE REVIEW

File: v0.7.0/concepts/mev-commit-chain/bridging-details.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ There are inherent security assumptions in bridging ETH to the mev-commit chain.
 - POA signers that maintain mev-commit chain state.
 - Correctness of Primev's [standard bridge protocol](https://github.com/primev/mev-commit-bridge/tree/main/standard/bridge-v1) and integration into the mev-commit chain.
 
-Our standard bridge implementation is currently being audited and has always been open source.
+Our standard bridge implementation has been audited by Canita. More details can be found [here](https://blog.primev.xyz/Battle-Testing-mev-commit-Key-Findings-from-Our-Spearbit-Security-Review-1436865efd6f8025bd6ce3824349821d).
```
Comment (aloknerurkar) [inline v0.7.0/concepts/mev-commit-chain/bridging-details.mdx:24]: Typo: cantina?

File: v0.7.0/get-started/bridge.mdx [hunk 2]
```diff
@@ -6,6 +6,15 @@ icon: "bridge"
 
 <p> The mev-commit chain uses the native ETH tokens that need to be bridged from L1. We use a lock and mint bridge. A user CLI application can be downloaded from the [releases page](https://github.com/primev/mev-commit/releases/latest). </p>
 
+Or download the 0.7.0 bridge CLI directly from the links below:
```
Comment (aloknerurkar) [inline v0.7.0/get-started/bridge.mdx:9]: We will be doing a patch release before the next mainnet release. Possibly v0.7.1. Do we need to update this?

**Review by aloknerurkar — APPROVED** [review]

File: v0.7.0/concepts/mev-commit-chain/bridging-details.mdx (hunk 1 above)
Comment (shaspitz) [reply v0.7.0/concepts/mev-commit-chain/bridging-details.mdx:24]: Yea that's the same organization as spearbit, will change if it wasn't clear to you

File: v0.7.0/get-started/bridge.mdx (hunk 2 above)
Comment (shaspitz) [reply v0.7.0/get-started/bridge.mdx:9]: Yea we should likely just update this bullet point list once the release actually happens. This was a request from murat that we make it easier for people to download the binary directly with one click

**Pull request merged**